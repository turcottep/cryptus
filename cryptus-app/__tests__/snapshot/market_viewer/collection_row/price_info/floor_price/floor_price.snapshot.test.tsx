import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

import FloorPrice from "../../../../../../components/market_viewer/collection_row/price_info/floor_price/floor_price";

describe("<FloorPrice />", () => {
  it("should display properly", async () => {
    const test_price = 7.79;
    const test_curr = "eth";

    const tree = renderer
      .create(<FloorPrice price={test_price} currency={test_curr} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
