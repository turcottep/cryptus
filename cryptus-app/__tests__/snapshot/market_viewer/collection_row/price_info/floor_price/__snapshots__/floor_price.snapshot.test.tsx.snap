// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FloorPrice /> should display properly 1`] = `
<div>
  7.8Îž
</div>
`;
